app/src/i18n/locales/de/settings.json

```diff
@@ -49,5 +49,15 @@
     "delete-file": "Datei löschen",
     "status": "Status",
     "download-csv": "Als CSV herunterladen",
-    "delete-inventory": "Inventar löschen"
+    "delete-inventory": "Inventar löschen",
+    "delete-inventory-prompt": "Sind Sie sicher, dass Sie das Inventar dieser Stadt <2>dauerhaft löschen</2> möchten?",
+    "enter-password-info": "Geben Sie Ihr Passwort ein, um das Löschen der Städteinformationen zu bestätigen.",
+    "user-details-updated": "Benutzerdaten aktualisiert",
+    "something-went-wrong": "Etwas ist schief gelaufen",
+    "is-organization": "Ist eine Organisation",
```

Comment:
This will be removed from the user model in my upcoming PR btw.
So we can get rid of the checkbox for it (either here or in the future).